test-preprocessor.js

```diff
@@ -0,0 +1,11 @@
+const tsc = require('typescript');
+const tsConfig = require('./tsconfig.json');
+
+module.exports = {
+    process(src, path) {
```

Comment:
Please configure your IDE to use 2 spaces for indentation, and change the indentation in other files.